source/_components/homematicip_cloud.markdown

```diff
@@ -14,31 +14,67 @@ ha_release: 0.66
 featured: false
 ---
 
+## HomematicIP
+
 The [HomematicIP](http://www.homematic-ip.com) component platform is used as an interface to the cloud server.
-For for communication [homematicip-rest-api](https://github.com/coreGreenberet/homematicip-rest-api) is used.
+For for communication [homematicip-rest-api](https://github.com/coreGreenberet/homematicip-rest-api) is used. Since there is no official documentation about this API everything was done via reverse engineering. Use at your own risk.
 
-To set up the component:
+### Setup the component via interface 
 
-- generate the authentication token:
-```yaml
-generate_auth_token.py
-```
+Menu: *Configuration* -> *Integrations*
+  
+Fill the form:
+* your **access point ID** (SGTIN)
+* Optional a **name** to identify your access point, this will be used to prefix your device names.
+  
+The authentification token will be generated and stored internaly.
 
-- add the information to your `configuration.yaml` file:
-```yaml
+### Setup the component via `configuration.yaml`:
+  
+Generate the authentication token:
+  
+`generate_auth_token.py`
+  
+Add the information to your `configuration.yaml` file:
+
+```    
 homematicip_cloud:
   - accesspoint: IDENTIFIER
     authtoken: AUTHTOKEN
   - name: Location2
     accesspoint: IDENTIFIER2
-    authtoken: AUTHTOKEN2   
+    authtoken: AUTHTOKEN2
 ```
 
 Configuration variables (global):
+* **name**: (*Optional*): Name to identify your access point, this will be used to prefix your device names.
+* **accesspoint**: (*Required*): This is the access point id (SGTIN)
+* **authtoken**: (*Required*): Authentification token generated with `generate_auth_token.py`.
 
-- **name** (*Optional*): Name to identify your access point, this will be
-  used to prefix your device names.
-- **accesspoint** (*Required*): This is the access point id (SGTIN)
-- **authtoken** (*Required*): Authentification token generated with
-`generate_auth_token.py`.
-
+### Actual implemented and tested
+  * homematicip_cloud.alarm_control_panel
+    - [x] Security zones (*HmIP-SecurityZone*)
```

Comment:
Can you remove the `[x]` 

https://deploy-preview-5966--home-assistant-docs.netlify.com/components/homematicip_cloud/